modules/synthetics/variables.tf

```diff
@@ -21,3 +21,9 @@ variable "locations" {
   description = "Array of locations used to run synthetic tests"
   default     = []
 }
+
+variable "public_locations" {
```

Comment:
we have variable "locations" and variable "public_locations" now, please make it one var